### ISSUE TITLE
Feature Request : changing ads id after initialise

### ISSUE THREAD

**Issue by nickgoh** [bdlukaa/native_admob_flutter#2]

**Comment by bdlukaa** [comment]
You can do that. Use `MobileAds.bannerAdUnitId` (Or whatever id you want), for example:

```dart
await MobileAds.initialize();
MobileAds.interstitialAdUnitId = 'ca-unit-xxxxxxxxxx-xxxxxxxxxx'
```

I will make sure to mention that in the documentation

You can also change set the unit id in the load methods of some ads:
  - Native
  - AppOpen

Setting the unit id in the load methods will be avaiable to Interstitial and Rewarded ads once I migrate to [Android SDK v20](https://developers.google.com/admob/android/migration)

**Comment by bdlukaa** [comment]
I just published a new version: [0.7.0](https://pub.dev/packages/native_admob_flutter). You can now set the unit id in the load methods of:
- Native
- AppOpen
- Interstitial
- Rewarded

Support for this on BannerAds is planned for the next version!
